File: Java/Liczby Pierwsze.java
import java.util.Scanner;

public class Main 
{
    public static boolean isPrime(int number) 
    {
        if (number <= 1)
            return false;
        for (int i = 2; i <= Math.sqrt(number); ++i)
            if (number % i == 0)
                return false;
        return true;
    }

    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        for (int i = 0; i < n; i++) 
        {
            int number = input.nextInt();
            if (isPrime(number))
                System.out.println("TAK");
            else
                System.out.println("NIE");
        }
        input.close();
    }
}

File: Python/Liczby Pierwsze.py
def is_prime(number):
    if number < 2:
        return 'NIE'
    for i in range(2, int(number ** 0.5) + 1):
        if number % i == 0:
            return 'NIE'
    return 'TAK'

n = int(input())
for _ in range(n):
    number = int(input())
    print(is_prime(number))